core/src/main/java/tachyon/master/MasterInfo.java

```diff
@@ -543,6 +547,8 @@ int _createFile(boolean recursive, TachyonURI path, boolean directory, long bloc
         mFileIdToInodes.put(dir.getId(), dir);
         currentInodeFolder = (InodeFolder) dir;
       }
+      mMasterSource.incFilesCreated(parentPath.length - pathIndex);
+      mMasterSource.incCreateFileOps();
```

Comment:
Why is `incCreateFileOps()` called here instead of the beginning (attempts) or end (successful attempts) of the function?